### ISSUE TITLE
Tab icons not displayed correctly using bootstrap

### ISSUE THREAD

**Issue by carlos-montiers** [joeattardi/emoji-button#33]
Hello. Great component.
Using this with angular 8 and bootstrap 4 I that the tab icons are not showed correctly, like this:
![image](https://user-images.githubusercontent.com/5741891/75361429-09254e80-5896-11ea-9ee3-d315d2b8ae31.png)

This is caused by _reboot.scss:
```
*, ::after, ::before {
box-sizing: border-box;
}
```

I fixed this issue with this css:
```
.emoji-picker__tab svg {
    box-sizing: content-box;
}
```

I shared it for maybe include that in the default css in next versions.

**Comment by joeattardi** [comment]
Thanks for finding this! I will definitely incorporate this into the next release.

**Comment by joeattardi** [comment]
I have just released version 2.11.0 which contains your CSS change.
